lib/useBackboneContext.js

```diff
@@ -0,0 +1,18 @@
+const {useContext} = require('react');
+const Context = require('./context');
+
+function useBackboneContext() {
+  const backboneContext = useContext(Context);
+
+  if (backboneContext === undefined) {
```

Comment:
We can actually let this gracefully degrade and use the default value of context. [Docs here:](https://reactjs.org/docs/context.html#reactcreatecontext)

> The defaultValue argument is only used when a component does not have a matching Provider above it in the tree. This can be helpful for testing components in isolation without wrapping them. Note: passing undefined as a Provider value does not cause consuming components to use defaultValue.

To that end we should make the default value an empty object to keep that contract the same no matter if there's a provider or not.